eitprocessing/categories.py

```diff
@@ -208,6 +210,44 @@ def get_default_categories() -> Category:
         return Category.from_compact_yaml(fh.read())
 
 
+def check_category(data: DataContainer, category: str, *, raise_: bool = False) -> bool:
+    """Check whether the category of a dataset is a given category or one of it's subcategories.
+
+    Example:
+    >>> data = ContinuousData(..., category="impedance", ...)
+    >>> check_category(data, "impedance")  # True
+    >>> check_category(data, "pressure")  # False
+    >>> check_category(data, "pressure", raise_=True)  # raises ValueError
+    >>> check_category(data, "does not exist", raise_=False)  # raises ValuError
+
+    Args:
+        data: DataContainer object with a `category` attribute.
+        category: Category that the object should be (a subcategory of).
```

Comment:
I'm unsure of the purpose of the parentheses in this sentence. Is it supposed to be (sub)category?